plugins/login-plugin/src/Components/UIComponents/Storefront/StoreFrontTv/FeesScrollView/index.js

```diff
@@ -0,0 +1,47 @@
+import React from "react";
+// import {
+//   SafeAreaView,
+//   View,
+//   FlatList,
+//   StyleSheet,
+//   Text,
+//   StatusBar,
+// } from "react-native";
+
+import { StyleSheet, ScrollView } from "react-native";
+import { mapKeyToStyle } from "../../../../Utils/Customization";
+import FeeCard from "./FeeCard";
+FeesScrollView.propTypes = {
+  screenStyles: {},
+  payload: { extensions: {} },
+};
+
+const Item = ({ title }) => (
+  <View style={styles.item}>
+    <Text style={styles.title}>{title}</Text>
+  </View>
+);
+
+const FeesScrollView = (props) => {
+  const {
+    screenStyles,
+    payload: { extensions = {} },
+  } = props;
+
+  const renderItem = ({ item }) => <Item title={item.title} />;
```

Comment:
please use callback hook for rendering to avoid unnecessary re-renders. 
```suggestion
  const renderItem = React.useCallback(({ item }) => <Item title={item.title} />, [item.title]);
```